Sources/SwiftDriver/Jobs/FrontendJobHelpers.swift

```diff
@@ -180,6 +180,15 @@ extension Driver {
     try commandLine.appendAllArguments(.Xfrontend, from: &parsedOptions)
     try commandLine.appendAll(.coveragePrefixMap, from: &parsedOptions)
 
+    // Pass down -user-module-version if we are working with a compiler that
+    // supports it.
+    if let ver = parsedOptions.getLastArgument(.userModuleVersion)?.asSingle {
+      if isFrontendArgSupported(.userModuleVersion) {
```

Comment:
```suggestion
    if let ver = parsedOptions.getLastArgument(.userModuleVersion)?.asSingle,
       isFrontendArgSupported(.userModuleVersion) {
```
Can be one if statement